app/move/controllers/create/document.js

```diff
@@ -17,6 +18,16 @@ class DocumentUploadController extends CreateBaseController {
     this.use(this.populateDocumentUpload)
   }
 
+  checkFeatureEnabled(enabled) {
+    return function functionName(req, res, next) {
```

Comment:
Perhaps use a function expression rather than giving it a generic name?

Reply:
Doh! That was me trying to figure out a failing test. Good spot :)